src/parser/csql_lexer.l

```diff
@@ -580,6 +580,8 @@ int yybuffer_pos;
 [pP][aA][gG][eE]							{ begin_token(yytext);
 										csql_yylval.cptr = pt_makename(yytext);
 										return PAGE; }
+[pP][aA][lL][iI][nN][dD][rR][oO][mM][eE]				{ begin_token(yytext); csql_yylval.cptr = pt_makename(yytext);
```

Comment:
We have a 120 character line limit, please set `csql_yylval.cptr = pt_makename(yytext);` on the next line.